vic/drivers/shared_image/src/vic_store.c

```diff
@@ -1324,9 +1329,13 @@ set_nc_state_file_info(nc_file_struct *nc_state_file)
     nc_state_file->time_size = NC_UNLIMITED;
     nc_state_file->veg_size = options.NVEGTYPES;
 
+    // set ids and dimension sizes of the extension variables
+    set_nc_state_file_info_extension(nc_state_file);
```

Comment:
rename to `set_nc_state_file_info_rout_extension`

Reply:
done